packages/clickable/src/use-clickable.ts

```diff
@@ -99,16 +99,19 @@ export function useClickable(props: UseClickableProps = {}) {
     [isDisabled, onClick],
   )
 
-  const onDocumentKeyUp = (e: KeyboardEvent) => {
-    if (isPressed && isValidElement(e)) {
-      e.preventDefault()
-      e.stopPropagation()
+  const onDocumentKeyUp = React.useCallback(
```

Comment:
This is a question: 
Why in every component project in everything the members use React functions in this way `React.useCallback`?
And why not import these functions in from `import { useCallback, useMemo} from 'react'`?

Reply:
Hi @jvmazagao! 👋 

Good questions!

1. We should probably be using `React.useCallback` more often, but it isn't necessary to use them in all cases. They're useful when you need referential transparency (for example, they're being used as dependencies to a `useEffect` or another `useCallback`) and avoiding computationally expensive calculations. If you don't need either of those things, you're actually reducing performance for no gain.

Kent C. Dodds [has a good post on this topic](https://kentcdodds.com/blog/usememo-and-usecallback) that I recommend reading.

2. It's just a style choice. They're equivalent. I like to use `React.useCallback` so it's clear we're using an official hook, but it's just as valid to important and use `useCallback` by itself.

Reply:
Thanks, @with-heart I'll take a look at the article, and I here to review some PR's, every time.

Reply:
@jvmazagao No problem! Please feel free to ask questions and give feedback 😁 The more the merrier!